### PR TITLE
CLI interface : Added boot command

### PR DESCRIPTION
**Pull request by in1004kyu**
Add boot command at guestloader.
It provides command "help", "boot", "status", "banner"
help(h) : List commands and their usage
boot(b) : Boot guest
status(s) : Showing hypervisor status (not implemented yet).

### CODE REVIEW

**Comment by kesl** [comment]
CI : Test FAILed.
Refer to this link for build results: http://keslproject.iptime.org:8082/job/k-hypervisor_pull_request/122/

File: platform-device/cortex_a15x2_rtsm/guestos/guestloader/drivers/pl01x.c [hunk 1]
```diff
@@ -0,0 +1,111 @@
+#include "pl01x.h"
```
Comment (in1004kyu) [inline platform-device/cortex_a15x2_rtsm/guestos/guestloader/drivers/pl01x.c:1]: I will move this file to platform-device/cortex_a15x2_rtsm/drivers/uart/

**Comment by joohyun0115** [comment]
retest this please

**Comment by kesl** [comment]
CI : Test FAILed.
Refer to this link for build results: http://keslproject.iptime.org:8082/job/k-hypervisor_pull_request/123/

**Comment by joohyun0115** [comment]
We will automatically boot into the OS using timer, please change that.

**Comment by in1004kyu** [comment]
Ok~

**Comment by Wonseok** [comment]
Rename overall names such as function, macro, variables.

**Pull request closed without merging**